static/js/billing/upgrade.js

```diff
@@ -33,12 +37,38 @@ exports.initialize = () => {
         e.preventDefault();
     });
 
+    $("#start-free-trial-button").on("click", (e) => {
+        e.preventDefault();
+        let redirect_url = "/billing";
+        if ($("#freetrial-form").data("onboarding")) {
+            redirect_url = "/billing?onboarding=true";
+        }
+        helpers.create_ajax_request(
+            "/json/billing/upgrade",
+            "freetrial",
+            undefined,
+            [],
+            redirect_url,
+        );
+    });
+
+    $("#start-on-zulip-free-button").on("click", (e) => {
+        e.preventDefault();
+        window.location.replace("/");
+    });
+
     $("#invoice-button").on("click", (e) => {
         if (helpers.is_valid_input($("#invoiced_licenses")) === false) {
             return;
         }
         e.preventDefault();
-        helpers.create_ajax_request("/json/billing/upgrade", "invoice", undefined, ["licenses"]);
+        helpers.create_ajax_request(
+            "/json/billing/upgrade",
+            "invoice",
+            undefined,
+            ["licenses"],
+            "",
```

Comment:
Why is the `redirect_to` argument value changing here?

Reply:
I extracted it as a different commit. It was not really required in that commit.

https://github.com/zulip/zulip/pull/16101/commits/c7d5009f237dd233f9342cf0e5a632febd261abc